langwatch/src/server/api/routers/triggers.ts

```diff
@@ -0,0 +1,112 @@
+import { z } from "zod";
+import { createTRPCRouter, protectedProcedure } from "../trpc";
+import { TriggerAction } from "@prisma/client";
+import { TRPCError } from "@trpc/server";
+
+import { nanoid } from "nanoid";
+import { TeamRoleGroup, checkUserPermissionForProject } from "../permission";
+
+export const triggerRouter = createTRPCRouter({
+  create: protectedProcedure
+    .input(
+      z.object({
+        projectId: z.string(),
+        name: z.string(),
+        members: z.string().array(),
+        action: z.nativeEnum(TriggerAction),
+        actionParams: z.any(),
+        filters: z.any(),
+        organizationId: z.string(),
```

Comment:
nope, please remove the organizationId from here, it is not checked on the permissions, so I can actually pass in any organizationId I want. The projectId is already checked, so we can use that to navigate back to teams, where I think we should actually take the emails from (TeamUser) because maybe it would be a bit weird to send an email to somebody else on the org from another team wouldn't it?